app/services/analytics_events.rb

```diff
@@ -2406,6 +2406,31 @@ def idv_in_person_usps_proofing_results_job_enrollment_failure(reason:, enrollme
     )
   end
 
+  # Tracks if USPS in-person proofing enrollment request fails
+  # @param [String] context
+  # @param [String] reason
+  # @param [String] enrollment_id
```

Comment:
Would this be a number?

```suggestion
  # @param [Integer] enrollment_id
```

Reply:
Yep, updated in [946262f](https://github.com/18F/identity-idp/pull/6629/commits/946262f9b47195eb082f99a59f519748357aa969)